test/chainweb/pact/command_payload_test.exs

```diff
@@ -658,7 +663,11 @@ defmodule Kadena.Chainweb.Pact.CommandPayloadTest do
       pub_key: pub_key,
       signer_scheme: signer_scheme
     } do
-      cap = Cap.new(%{name: "coin.GAS", args: [["9007199254740992.553"]]})
+      cap =
+        Cap.new(%{
```

Comment:
Maybe this could look better like this 🤔 ?(probably changing the variable name) What do you think? @miguelnietoa 
```suggestion
        pact_values = [PactValue.new("9007199254740992.553")]
        Cap.new(%{
          name: "coin.GAS",
          args: [PactValue.new(pact_values)]
        })
```